db/migrate/20210308092402_add_os_version_to_profiles.rb

```diff
@@ -0,0 +1,18 @@
+class AddOsVersionToProfiles < ActiveRecord::Migration[5.2]
+  def up
+    remove_index :profiles, name: 'uniqueness'
+    add_column :profiles, :os_version, :string, default: nil
+    add_index :profiles,
+              %i[ref_id account_id benchmark_id external os_version policy_id],
+              unique: true, name: 'uniqueness'
+    OsVersionUpdater.run!
```

Comment:
We can try to separate schema migration and data migration, if that is even possible.

Reply:
Agreed. Let's add the os_version field with a default value of `''` and `null: false` and update existing profiles in another migration.